packages/jupyter-ai/schema/plugin.json

```diff
@@ -13,6 +13,13 @@
       }
     ]
   },
+  "jupyter.lab.shortcuts": [
+    {
+      "command": "jupyter-ai:focus-chat-input",
+      "keys": ["Accel Shift 1"],
```

Comment:
What is the `Accel` key? Is this `Ctrl`?

Reply:
"Command/Ctrl" ("It depends") see https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/getModifierState#accel_virtual_modifier (ignore deprecation warning, lumino re-implements it to avoid browser compatibility issues)

Also:
- https://jupyterlab.readthedocs.io/en/latest/extension/extension_points.html#keyboard-shortcuts
- https://jupyterlab.readthedocs.io/en/latest/developer/contributing.html#keyboard-shortcuts

Reply:
Oh, I see. Makes sense why this is given a unique name.